Tasks/PerformActionOnMachineGroup/task.json

```diff
@@ -16,7 +16,15 @@
         "Patch": 6
     },
     "minimumAgentVersion": "1.83.0",
-    "inputs": [      
+    "inputs": [ 
+        {
+            "name": "ConnectedServiceName",
+            "type": "connectedService:Azure",
+            "label": "Azure Subscription",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Azure subscription under which the Resource Group is deployed."
```

Comment:
Can you specify in the help down that ConnectedServiceName is not mandatory and can be empty in case of standard environment.